### PR TITLE
Use CC error info in MC

### PR DESCRIPTION
**Pull request by arukuka**
# Summary

- MC become able to return CC error

# Purpose

- When an exception was occured in CC,  
  client side could not know such a status.
- This patch will solve this issue partially.

# Contents

- Return detailed error information if it is exists
- Update protobuf generated code

# TODO

- implement
  - [x] MC UnitTest
  - [x] LibC IntegrationTest

# Testing Methods Performed

- Run demo with https://github.com/acompany-develop/QuickMPC-libClient-py/pull/21
- Run Added Test

### CODE REVIEW

File: src/ManageContainer/Client/ManageToDb/Client.go [hunk 1]
```diff
@@ -147,33 +149,67 @@ func (c Client) GetSchema(dataID string) ([]string, error) {
...
+
+		if errors.Is(err, errorSpecificStatusFileNotFound) {
+			continue
+		}
+
+		if err != nil {
+			return status, nil, err
 		}
+
+		return status, nil, nil
```
Comment (arukuka) [inline src/ManageContainer/Client/ManageToDb/Client.go:188]:
```suggestion
		return status, nil, err
```

File: src/ManageContainer/Server/LibcToManageContainer/L2m.go [hunk 2]
```diff
@@ -274,6 +273,14 @@ func (s *server) GetComputationResult(in *pb.GetComputationResultRequest, stream
 		return err
 	}
 
+	if computationErrInfo != nil {
+		status, err := status.New(codes.Unknown, "computation result has error info").WithDetails(computationErrInfo)
+		if err != nil {
+			return err
+		}
+		return status.Err()
+	}
```
Comment (arukuka) [inline src/ManageContainer/Server/LibcToManageContainer/L2m.go:282]: I found the other good API, so I used it.

**Comment by arukuka** [comment]
@mdonaka I added MC UnitTest, libC end2end test.  
I would appreciate it if you could review this PR again.

**Comment by arukuka** [comment]
Sorry, I forgot to rebase.  I will.

**Review by mdonaka — APPROVED** [review]
> @mdonaka I added MC UnitTest, libC end2end test.
I would appreciate it if you could review this PR again.

I checked it! Looks good! Thanks!

**Pull request merged**